tensor2tensor/data_generators/mathematical_language_understanding.py

```diff
@@ -52,9 +52,8 @@
 
 import tensorflow as tf
 
-
 @registry.register_problem
-class AlgorithmicMathTwoVariables(text_problems.Text2TextProblem):
+class MathematicalLanguageUnderstanding(text_problems.Text2TextProblem):
```

Comment:
Please keep the name as before, this is an algorithmic problem which makes it easier to locate in the large number of T2T datasets.